Wikipedia/Code/MWKSection.m

```diff
@@ -280,7 +280,7 @@ - (NSString*)summary {
 - (nullable NSArray<MWKTitle*>*)disambiguationTitles {
     NSArray* textNodes = [self elementsInTextMatchingXPath:WMFSectionDisambiguationTitlesXPathSelector];
     return [textNodes wmf_mapAndRejectNil:^id (TFHppleElement* node) {
-        if (node.text.length == 0 || [node.text containsString:@"redlink=1"]) {
+        if (node.text.length == 0 || [node.text containsString:@"redlink=1"] || [node.text containsString:@"cite_note"]) {
```

Comment:
not clear why we're combining XPath & manual string matching.  @coreyfloyd could you try consolidating this into the XPath, e.g. `//div[@class='hatnote']//a/[@href and not(@id = 'cite_note') and not(boolean(@redlink))]` @montehurd any thoughts?